### DISCUSSION TITLE
Did the linkerd2-multicluster helm chart stop getting updated?

### DISCUSSION THREAD

**Discussion by antifuchs** [linkerd/linkerd2#7214] [Q&A] [answered]
I'm trying to install linkerd2 into our clusters using helm (as that seems to be the most straightforward way to get it working with pulumi/terraform), and am hitting a wall. I installed the `linkerd2-cni` and `linkerd2` helm charts at version 2.11.1; however, the `linkerd2-multicluster` helm chart isn't available at that version.

The latest version I can find for it on [artifacthub](https://artifacthub.io/packages/helm/linkerd2/linkerd2) is 2.9.5, which seems incompatible: Installing that in my cluster results in a failing `linkerd multicluster check` run that claims:

```console
:;    linkerd multicluster check
namespace "multicluster" not found; install by running `linkerd multicluster install | kubectl apply -f -`
```

(and indeed, the multicluster namespace that gets installed is named `linkerd-multicluster`.)

Are there no new releases of this chart anymore, or did it get deprecated & the docs didn't get updated?

**Comment by alpeb** [comment, marked as answer]
Ah sorry about that, we failed to properly document this change, which is simply that the chart is now named `linkerd-multicluster`. You can find it here on version 2.11.1:
https://artifacthub.io/packages/helm/linkerd2/linkerd-multicluster

**Reply by navpreet-securitas** [reply]
Hi @alpeb  
i used `linkerd-multicluster` chart version 2.11.1 with installNamespace set to false. 
As linkerd-multicluster namespace was created before helmrelease(we are using fluxv2 to deploy). Helmrelease goes well, resources are deployed but `linkerd multicluster check` gives same error:

> namespace "multicluster" not found; install by running `linkerd multicluster install | kubectl apply -f -`

if we don't create namespace manually and set installNamespace to true, helm release creates namespace for us and then `linkerd multicluster check` succeeds .

only difference between namespace created by chart and created manually was in labels:
```
labels:
    linkerd.io/extension: multicluster
```
this was present with ns created by chart.

**Reply by alpeb** [reply]
Indeed, that label is required for `linkerd multicluster check` to identify which namespace is the multicluster one. If you're creating the namespace outside of the `linkerd-multicluster` chart you need to have your CD pipeline to account for that. Note that in the upcoming 2.12 we have stopped relying on that metadata in namespaces to better align with the Helm best-practice of allowing other tools to take the responsibility of creating the namespaces (also currently fixed in the latest edge releases).
